addons/test_mail/tests/test_message_track.py

```diff
@@ -39,6 +43,77 @@ def test_message_track_author(self):
 
         self.assertEqual(self.record.message_ids.author_id, self.partner_admin)
 
+    def test_message_track_default_message(self):
```

Comment:
Would use a '@users' to have ACLs activated :)  + don't forget docstring explaining we test '_track_get_default_log_message'